docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,470 @@
+    "# primitive, in a single batch so that the jobs will run back-to-back.\n",
+    "with Batch(backend=backend) as batch:\n",
+    "    sampler = SamplerV2(mode=batch)\n",
+    "    jobs = {\n",
+    "        label: sampler.run(subsystem_subexpts, shots=4096 * 3)\n",
+    "        for label, subsystem_subexpts in isa_subexperiments.items()\n",
+    "    }\n",
+    "\n",
+    "# Retrieve results\n",
+    "results = {label: job.result() for label, job in jobs.items()}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64e762ad-0660-4c78-8ea7-fb9d226357d7",
+   "metadata": {},
+   "source": [
+    "Lastly, the expectation value of the full circuit is reconstructed using the [`reconstruct_expectation_values`](/api/qiskit-addon-cutting/qiskit-addon-cutting#reconstruct_expectation_values) method.\n",
+    "\n",
+    "The code block below reconstructs the results and compares them with the exact expectation value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "b3cf1b65-df16-4bd4-a083-f65cec6c49dc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reconstructed expectation value: 0.5725264\n",
+      "Exact expectation value: 0.56254612\n",
+      "Error in estimation: 0.00998027\n",
+      "Relative error in estimation: 0.01774125\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get expectation values for each observable term\n",
+    "reconstructed_expval_terms = reconstruct_expectation_values(\n",
+    "    results,\n",
+    "    coefficients,\n",
+    "    subobservables,\n",
+    ")\n",
+    "\n",
+    "# Reconstruct final expectation value\n",
+    "reconstructed_expval = np.dot(reconstructed_expval_terms, observable.coeffs)\n",
+    "\n",
+    "\n",
+    "estimator = EstimatorV2()\n",
+    "exact_expval = (\n",
+    "    estimator.run([(qc, observable, [0.4] * len(qc.parameters))])\n",
+    "    .result()[0]\n",
+    "    .data.evs\n",
+    ")\n",
+    "print(\n",
+    "    f\"Reconstructed expectation value: {np.real(np.round(reconstructed_expval, 8))}\"\n",
+    ")\n",
+    "print(f\"Exact expectation value: {np.round(exact_expval, 8)}\")\n",
+    "print(\n",
+    "    f\"Error in estimation: {np.real(np.round(reconstructed_expval-exact_expval, 8))}\"\n",
+    ")\n",
+    "print(\n",
+    "    f\"Relative error in estimation: {np.real(np.round((reconstructed_expval-exact_expval) / exact_expval, 8))}\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "faf4620d-6782-4fce-a99b-9e1d01f29951",
+   "metadata": {},
+   "source": [
+    "## Next steps\n",
+    "\n",
+    "<Admonition type=\"tip\" title=\"Recommendations\">\n",
+    "    - Read the [Get started with circuit cutting using wire cuts](/guides/qiskit-addons-cutting-wires) guide.\n",
+    "    - Read the arXiv paper on [circuit knitting with classical communication.](https://arxiv.org/abs/2205.00016)\n",
```

Comment:
Update this with the version in the journal: https://ieeexplore.ieee.org/document/10236453